docs/ota-updates.md

```diff
@@ -0,0 +1,51 @@
+# Over The Air Unattended Updates (OTAU2)
+
+This document enumerates some of the approaches and FLOSS software that could
+be used to deploy OTAU2 to Lepidopter distribution.
+
+## Requirements
+
+A list of Lepidopter's OTAU2 requirements:
+
+* Atomic software release update
+
+* On failure, deploy previous working bootloader, kernel
+    configuration, and filesystems
```

Comment:
As we've discussed on IRC, it may be quite safe to _burn_ partition table, `/boot` and `/`. These three blobs take first ~1Gb of the card and remaining data may be left uninitialised. Moreover, initialization of the `/data` on boot may be part of wipe-on-failure strategy.